provider/resource_keycloak_realm.go

```diff
@@ -229,11 +229,47 @@ func resourceKeycloakRealm() *schema.Resource {
 				Computed:         true,
 				DiffSuppressFunc: suppressDurationStringDiff,
 			},
+
+			//internationalization
+			"internationalization": {
+				Type:     schema.TypeSet,
```

Comment:
Could you use `TypeList` like I did in #128?

Reply:
done